app/assets/stylesheets/components/_side-nav.scss

```diff
@@ -0,0 +1,10 @@
+.sidenav-footer {
+  border-top: solid 2px #112e51;
```

Comment:
For this, I'd recommend using the `$color-primary-darkest` Sass variable from the [Web Design Standards](https://standards.usa.gov/components/colors/):

```scss
border-top: 2px solid $$color-primary-darkest;
```

Reply:
I was just reading your pull req code changes and found this file :) 
https://github.com/jgarber623/uswds-rails/blob/ec0f4c7c71c359cacc68140a1c21b2f46a26f108/vendor/assets/stylesheets/uswds/core/_variables.scss